app/src/main/java/com/android/sample/ui/profile/Profile.kt

```diff
@@ -76,13 +91,24 @@ fun ProfileScreen(user: ProfileData, navigationAction: NavigationActions) {
                     // Handle navigation icon click
                   }) {
                     Icon(
-                        painter = painterResource(id = R.drawable.settings),
-                        modifier = Modifier.size(28.dp),
-                        tint = Color.Unspecified,
-                        contentDescription = "Settings")
+                        imageVector = Icons.Filled.Settings,
+                        contentDescription = "Settings",
+                        modifier =
+                            Modifier.size(28.dp).graphicsLayer(alpha = 0.99f).drawWithCache {
+                              onDrawWithContent {
+                                drawContent()
+                                drawRect(IconsGradientBrush, blendMode = BlendMode.SrcAtop)
+                              }
+                            })
                   }
             },
-        )
+            modifier =
+                Modifier.shadow(
+                        elevation = 2.dp, spotColor = ShadowColor, ambientColor = ShadowColor)
+                    .border(width = 1.dp, color = BorderColor)
+                    .width(412.dp)
+                    .height(48.dp)
+                    .background(color = PrimaryWhite))
```

Comment:
Consider using the colors from the MaterialTheme.colorScheme to maintain consistency.

```suggestion
                    .background(color = MaterialTheme.colorScheme.background))
```